atlp-rwanda/commanders-rn-medica
# Feature:Invite settings screen

**What does this PR do?**
This PR includes designs of inviting your friends to use the app

**Description of Task to be completed?**
Create a flatlist of some friends with their photos, names and phone numbers
create  an invite button where a user will click to invite his or her friends 

**How should this be manually tested?**
Click on the  Invite friends settings  on the first screen that appears and you will be directed to the invite setting screen where you can click on the invite button to invite your friends to this app

Any background context you want to provide?
N/A

What are the relevant pivotal tracker/Trello stories?
N/A

Screenshots

<img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/b1bd7a9f-f724-4804-bc33-5166099dd88f" width="250" />


<img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/a897db9d-7c03-4eeb-837b-c345884bfb2b" width="250" />
